composer.json

```diff
@@ -1,38 +1,57 @@
-{
-    "name": "thekeymaster/findologic-api",
-    "description": "Library for FINDOLOGIC API requests",
-    "type": "library",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Dominik Brader",
-            "email": "dominik@brader.co.at",
-            "role": "Maintainer"
-        },
-        {
-            "name": "Chris Ortner",
-            "email": "chris@codexfons.com",
-            "role": "Developer"
-        }
-    ],
-    "require": {
-        "guzzlehttp/guzzle": "^6.3",
-        "ext-SimpleXML": "*",
-        "ext-json": "*"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^4.8.35 || ^5.7 || ^6.4 || ^7.0",
-        "symfony/console": "v3.2.9",
-        "friendsofphp/php-cs-fixer": "^2.13"
-    },
-    "autoload": {
-        "psr-0": {
-            "FINDOLOGIC": "src/"
-        }
-    },
-    "autoload-dev": {
-        "psr-0": {
-            "FINDOLOGIC\\Request\\Tests": "tests/"
-        }
-    }
-}
+{
+    "name": "thekeymaster/findologic-api",
+    "description": "Library for FINDOLOGIC API requests",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Dominik Brader",
+            "email": "dominik@brader.co.at",
+            "role": "Maintainer"
+        },
+        {
+            "name": "Chris Ortner",
+            "email": "chris@codexfons.com",
+            "role": "Developer"
+        },
+        {
+            "name": "Markus Machatschek",
+            "email": "m.machatschek@findologic.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {

```

Comment:
I added this to the composer.json:

```json
"require": {
    "php": ">=5.6",
    "..."
}
```

Also while I was doing that I also tried to add `php 7.3` to the `.travis.yml`, but unfortunately `php-cs-fixer` has not yet introduced PHP 7.3 support. 

If it takes more time, I might set the maximum requirement of a php version to 7.2. But since PHP 7.3 is only stable since December, I prefer to wait some time before I limit the maximum requirements.